apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c

```diff
@@ -466,20 +312,11 @@ int buffer_optimization_main(int argc, char *argv[])
 	if (ret != OK) {
 		printf("Read test failed on file: %s, ret: %d\n", SAMPLE_FILE, ret);
 		ret = -1;
-		goto error_with_open_timer_dev;
 	}
 
-	ret = append_test(frt_fd, file_size);
-	if (ret != OK) {
-		printf("Append test failed on file: %s, ret: %d\n", SAMPLE_FILE, ret);
-		ret = -1;
-	}
-
-error_with_open_timer_dev:
 	if (ioctl(frt_fd, TCIOC_STOP, 0) < 0) {
 		fprintf(stderr, "ERROR: Failed to stop the Free Run Timer: %d\n", errno);
 		ret = -1;
-		goto error_with_frt_fd;
```

Comment:
Is this not required anymore ?

Reply:
Was initially used to skip the append test if the read test fails
We have now omitted the append test itself.